content/en/security/cloud_security_management/setup/agentless_scanning/cloudformation.md

```diff
@@ -63,10 +63,19 @@ If you've already [set up Cloud Security Management][3] and want to add a new AW
 
 To uninstall Agentless Scanning, log in to your AWS console and delete the CloudFormation stack created for Agentless Scanning.
 
+## Update the CloudFormation stack
+
+To update the Agentless Scanning CloudFormation stack:
+
+* Log in to your AWS console, select the `DatadogIntegration-DatadogAgentlessScanning...` CloudFormation sub-stack and click on `Update nested stack`
```

Comment:
I'd mention somewhere that this won't work if your version < 2.0.0. Updates on that earlier version are dangerous.